packages/calypso-e2e/src/lib/components/editor-sidebar-block-inserter-component.ts

```diff
@@ -78,10 +80,7 @@ export class EditorSidebarBlockInserterComponent {
 		if ( type === 'pattern' ) {
 			locator = editorParent.locator( selectors.patternResultItem( name ) ).first();
 		} else {
-			locator = editorParent
-				.getByRole( 'tabpanel', { name: 'Blocks' } )
-				.getByRole( 'option', { name, exact: true } )
-				.first();
+			locator = editorParent.locator( selectors.blockResultItem( name ) ).first();
```

Comment:
@dpasque I had to revert the changes here added in https://github.com/Automattic/wp-calypso/pull/81044. They were causing some tests to fail (run the GB simple tests on `trunk` to see them) because the block option couldn't be located for some reason. From what I see, it has to do with the fact the DOM structure changes slightly depending on how the option is filtered (i.e sometimes the tab buttons are not shown). I didn't dive deeply into it, though, so feel free to open another PR with a proper fix if you think it's appropriate :)